exchanges/yobit/yobit_wrapper.go

```diff
@@ -187,13 +187,14 @@ func (y *Yobit) UpdateTicker(p currency.Pair, assetType asset.Item) (ticker.Pric
 		return tickerPrice, err
 	}
 
-	for _, x := range y.GetEnabledPairs(assetType) {
-		curr := y.FormatExchangeCurrency(x, assetType).Lower().String()
+	enabledPairs := y.GetEnabledPairs(assetType)
+	for i := range enabledPairs {
+		curr := y.FormatExchangeCurrency(enabledPairs[i], assetType).Lower().String()
```

Comment:
This is out of scope but for extra SPEED we can do one map lookup instead of 7 (we've been cleaning this up as we go ever since your ws ticker/orderbook performance boost)